buckets-mdapi/src/main.rs

```diff
@@ -44,8 +44,12 @@ fn main() {
     );
 
     // Configure and start metrics server
+    // let metrics_config = config.metrics.clone();
```

Comment:
should this be removed?

Reply:
Ah yes, nice catch. Definitely a comment I meant to take out.